frontend/src/app/Settings/ManageOrganization.tsx

```diff
@@ -7,37 +7,10 @@ import RequiredMessage from "../commonComponents/RequiredMessage";
 import Alert from "../commonComponents/Alert";
 import Select from "../commonComponents/Select";
 import { showNotification } from "../utils";
+import { OrganizationTypeEnum } from "../signUp/Organization/utils";
 
 import { EditableOrganization } from "./ManageOrganizationContainer";
 
-const organizationTypes: { value: OrganizationType; label: string }[] = [
-  { value: "k12", label: "K-12 School" },
-  { value: "camp", label: "Camp" },
-  { value: "university", label: "College/University" },
-  { value: "correctional_facility", label: "Correctional Facility" },
-  { value: "employer", label: "Employer" },
-  { value: "government_agency", label: "Government Agency" },
-  { value: "airport", label: "Airport/Transit Station" },
-  { value: "shelter", label: "Homeless Shelter" },
-  { value: "fqhc", label: "FQHC" },
-  { value: "primary_care", label: "Primary Care / Mental Health Outpatient" },
-  { value: "assisted_living", label: "Assisted Living Facility" },
-  { value: "hospital", label: "Hospital or Clinic" },
-  { value: "urgent_care", label: "Urgent Care" },
-  { value: "nursing_home", label: "Nursing Home" },
-  { value: "treatment_center", label: "Substance Abuse Treatment Center" },
-  { value: "hospice", label: "Hospice" },
-  { value: "pharmacy", label: "Pharmacy" },
-  { value: "lab", label: "Lab" },
-  { value: "other", label: "Other" },
-];
-
-interface Props {
```

Comment:
Maybe deleted too much here? The app won't build for me because there  is no `Props` type

Reply:
Oh jeez, thank you!